utils/documentation_tests.txt

```diff
@@ -64,6 +64,7 @@ src/transformers/models/sew_d/modeling_sew_d.py
 src/transformers/models/speech_encoder_decoder/modeling_speech_encoder_decoder.py
 src/transformers/models/speech_to_text/modeling_speech_to_text.py
 src/transformers/models/speech_to_text_2/modeling_speech_to_text_2.py
+src/transformers/models/segformer/modeling_tf_segformer.py
```

Comment:
I see that the doc example for semantic segmentation doesn't actually test anything. 

You only test something if you include an expected output. We can include the expected output shape.

Reply:
Feel free to update the PyTorch one as well

Reply:
Both the semantic segmentation classes don't have the `add_code_sample_docstrings` decorator. Any suggestion on where to best include the expected output shape?

Reply:
Nevermind. Added something. Let me know.

Reply:
Yes looks good! Should be added for the PyTorch one as well

Reply:
I have added it. Refer here: https://github.com/huggingface/transformers/pull/17910/files#diff-3b11fdec06eefd9ffc22616fda87863b8bc2c54ea175c63ec08333490116861a